app/javascript/components/recipes/index/filters.vue

```diff
@@ -23,6 +23,13 @@
     >
       {{ filters[filter].min }} &lt; {{ $t(`msg.recipes.${filter}`) }} &lt; {{ filters[filter].max }}
     </div>
+    <!-- <div
+      v-if="showPriceFilter"
+      class="flex flex-row inline-block items-center justify-center px-3 py-3 static w-auto h-8
+      rounded-full flex-none flex-grow-0 mx-2.5 bg-yellow-500 text-white"
+    >
+    {{ filters[price].min }} &lt; {{ $t(`msg.recipes.price`) }} &lt; {{ filters[price].max }}
+    </div> -->
```

Comment:
en general evitamos dejar código comentado porque después no se sabe si se puede borrar o no y genera código inútil en los repos. Aparte por lo que veo es algo nuevo que no se usaba o sí?